tools/castle-editor/code/formchooseproject.pas

```diff
@@ -70,6 +73,24 @@ implementation
 
 { TChooseProjectForm ------------------------------------------------------------- }
 
+procedure TChooseProjectForm.Show;
+begin
+  {$IFDEF WINDOWS}
```

Comment:
We use `{$ifdef MSWINDOWS}` throughout the CGE code. I.e.
- Compiler directives are lowercase like `ifdef` not `IFDEF` (same note about `{$else}` and`{$endif}` below)
- We use `MSWINDOWS`, because that's also supported by Delphi. (FPC understands both `WINDOWS` and `MSWINDOWS`)

Reply:
Fixed in latest commit.